code/modules/xenomorph/xeno_structures.dm

```diff
@@ -42,6 +42,18 @@
 	SIGNAL_HANDLER
 	obj_destruction(damage_flag = "melee")
 
+/obj/structure/xeno/attack_alien(mob/living/carbon/xenomorph/X, damage_amount, damage_type, damage_flag, effects, armor_penetration, isrightclick)
```

Comment:
I dont see the point of this change

Reply:
So you don't end up with unremovable psi structures (like turrets) in valhalla.